modules/snooker/src/main/ts/ephox/snooker/model/RunOperation.ts

```diff
@@ -40,10 +41,12 @@ export interface TargetUnmergable {
   readonly unmergable: () => Option<Element[]>;
 }
 
-export interface TargetPaste {
+// combines the above 4 interfaces because this is what data we actually get from TinyMCE
+export interface CombinedTargets {
   readonly element: () => Element;
-  readonly generators: () => SimpleGenerators;
-  readonly clipboard: () => Element;
+  readonly mergable: () => Option<any>;
```

Comment:
I'm guessing it doesn't work with `ExtractMergable` instead of `any`, but wanted to flag it just in case

Reply:
Ah, it is `ExtractMergable`, yeah. Switching to extends which should fix that anyway.